packages/wallets/src/features/cashier/constants/constants.ts

```diff
@@ -49,49 +49,122 @@ export const walletIconMapper = {
     },
 } as const;
 
-export const appIconMapper = {
-    binary: {
-        dark: 'IcWalletOptionsDark',
-        light: 'IcWalletOptionsLight',
+export const LandingCompanyDetails = {
+    bvi: {
+        name: 'bvi',
+        shortcode: 'BVI',
     },
-    ctrader: {
-        dark: 'IcWalletCTrader',
-        light: 'IcWalletCTrader',
+    labuan: {
+        name: 'labuan',
+        shortcode: 'Labuan',
     },
-    derivez: {
-        dark: 'IcWalletDerivEZ',
-        light: 'IcWalletDerivEZ',
+    maltainvest: {
+        name: 'maltainvest',
+        shortcode: 'Maltainvest',
     },
-    dxtrade: {
-        dark: 'IcWalletDerivX',
-        light: 'IcWalletDerivX',
+    svg: {
+        name: 'svg',
+        shortcode: 'SVG',
     },
-    mt5: {
-        all: {
+    vanuatu: {
+        name: 'vanuatu',
+        shortcode: 'Vanuatu',
+    },
+} as const;
+
+export const MT5MarketTypeDetails = {
+    all: {
+        icon: {
             dark: 'IcWalletMt5All',
             light: 'IcWalletMt5All',
         },
-        financial: {
+        name: 'all',
+        title: 'MT5 Swap-Free',
+    },
+    financial: {
+        landingCompany: {
             malta: {
-                dark: 'IcWalletMt5CFDs',
-                light: 'IcWalletMt5CFDs',
+                icon: {
+                    dark: 'IcWalletMt5CFDs',
+                    light: 'IcWalletMt5CFDs',
+                },
+                name: 'malta',
+                title: 'MT5 CFDs',
             },
             svg: {
-                dark: 'IcWalletMt5Financial',
-                light: 'IcWalletMt5Financial',
+                icon: {
+                    dark: 'IcWalletMt5Financial',
+                    light: 'IcWalletMt5Financial',
+                },
+                name: 'svg',
+                title: 'MT5 Financial',
             },
             virtual: {
-                dark: 'IcWalletMt5Financial',
-                light: 'IcWalletMt5Financial',
+                icon: {
+                    dark: 'IcWalletMt5Financial',
+                    light: 'IcWalletMt5Financial',
+                },
+                name: 'virtual',
+                title: 'MT5 CFDs',
             },
         },
-        synthetic: {
+        name: 'financial',
+        title: 'MT5 Financial',
+    },
+    synthetic: {
+        icon: {
             dark: 'IcWalletMt5Derived',
             light: 'IcWalletMt5Derived',
         },
+        name: 'synthetic',
+        title: 'MT5 Derived',
+    },
+} as const;
+
+export const PlatformDetails = {
+    binary: {
+        icon: {
+            dark: 'IcWalletOptionsDark',
```

Comment:
I really like that we avoided concept of "mapper" as they tend to eventually became just a blob of data of all types. Its way better to have single responsibility objects, which ideally represents an actual real concept.